extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -263,6 +306,11 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
    * @private
    */
   onPlpCardClick_(shoppingData) {
+    this.variableService_.onVariableUpdate(
+      AnalyticsVariable.STORY_SHOPPING_PRODUCT_ID,
+      shoppingData?.productId
+    );
+    this.analyticsService_.triggerEvent(StoryAnalyticsEvent.SHOPPING_PDP_VIEW);
```

Comment:
See previous comment, this could be removed.

Reply:
Why was this comment resolved? There is no response to my comment :( 
I'm happy to clarify anything if there is any questions about my comments!

Reply:
I'll double check this one, it is likely that this code looked similar to other codeblocks, and unfortunately, the line numbers change when I update it - so I must've thought I covered it.